openlibrary/templates/trending.html

```diff
@@ -30,5 +30,7 @@ <h1>$_('Trending Books'): $(pages[mode])</h1>
   </div>
   <div class="pager">
     $:macros.Pager(page, 200, results_per_page=20)
+    $if mode == 'forever':
+      <p class="collection-date">Data Collection Started: October 2017</p>
```

Comment:
```suggestion
      <p class="collection-date">$_("Data Collection Started: October 2017")</p>
```
Please add the `$_()` so this string can be internationalized :) 
@etgibbs

Reply:
thanks, committed this change 👍 

curious about the process after reading this documentation [on internationalization](https://github.com/internetarchive/openlibrary/wiki#Internationalization) - should I have included the results of [this build](https://github.com/internetarchive/openlibrary/wiki#extracting-strings-from-htmlpython-files-generating-the-pot-file) and created a .pot file to include with this commit? still learning :)